dracut/99flatcar-debloat/module-setup.sh

```diff
@@ -0,0 +1,21 @@
+config() {
+    # gcrypt: Only needed for systemd-journal's FSS feature.
+    # lz4: Flatcar has never needed this for the journal or coredumps.
+    omit_dlopen_features+=" libsystemd-shared-*.so:gcrypt,lz4 "
+}
+
+# shellcheck disable=SC2064
+trap "$(shopt -p extglob)" RETURN
+shopt -q -s extglob
+
+install() {
+    # shellcheck disable=SC2064
+    trap "$(shopt -p extglob)" RETURN
+    shopt -q -s extglob
+
+    # Remove the NSS modules we don't need.
+    rm "${initdir}"/usr/lib*/libnss_!(dns|files|myhostname|resolve|systemd).so*
+
+    # We maybe should include this, but more work is needed for compliance.
+    rm "${initdir}"/usr/lib*/ossl-modules/fips.so
```

Comment:
This is probably something @tormath1 would be interested to see (if I remember correctly, you were working on fips mode in Flatcar, right?).

Reply:
I already spoke to him about it. :slightly_smiling_face: